test_infra/requirements.txt

```diff
@@ -0,0 +1,10 @@
+aws-cdk.core
```

Comment:
I am torn on this one:

1. I suppose it's better to split the aws-cdk requirements from our actual requirements-dev.txt but on the other it's a separate requirements file that we need to manage
2. I see that no version is specified. I think it makes sense otherwise dependabot would be constantly raising PRs but at the same time latest versions of the CDK libraries are more likely to have issues/bugs...

Reply:
On 1 - my main reasoning was that, for example, you may want to only run mocked tests and there is no reason for you to install CDK. I feel like most of the users don't go as far as spinning up their own test infrastructure.
On 2 - good catch, the version should definitely be fixed.

Reply:
1 makes perfect sense. For 2, it will be easier to manage with CDK v2 but for now I am not sure we want to fix the versions as dependabot will go wild haha